pkg/scanners/google/google.go

```diff
@@ -66,6 +64,9 @@ func validate(google *storage.GoogleConfig) error {
 	if google.GetProject() == "" {
 		errorList.AddString("ProjectID must be specified for Google Container Analysis")
 	}
+	if google.GetWifEnabled() {
+		errorList.AddString("Workload identities are not supported for Scanner integrations")
+	}
```

Comment:
Bit confused where this is now coming from, seems unrelated to the rest of the changes - care to add a comment _why_?

Reply:
Scanner and registries share the same image registration proto. We discussed that we don't want to support WIF for GCP scanners because they'll be deprecated/removed soon. That's why I added this check.